.github/workflows/icu_valgrind.yml

```diff
@@ -16,15 +16,15 @@ on:
 
 jobs:
   clang-valgrind-test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I explicitly use 20.04 before because somehow latest does not work. I just approve the run of the action and we could see is now working or not. If not, please revert this part of change.

Reply:
The Valgrind tests fail because the `ubuntu-latest` is currently running Ubuntu 22.04 that ships with Clang 14 which generates DWARF 5 instead of 4, as Valgrind only supports reading DWARF 4 at the moment, I'm going to use the `-gdwarf-4` compiler flag to generate DWARF 4 to see how it goes.

Reply:
thank you for helping us figure out why valgrind does not work in the latest. I approve the run again and let see what happened.